packages/sdk-ethers/src/index.ts

```diff
@@ -10,28 +10,46 @@ export const erc6551RegistryAddress =
 export const erc6551AccountImplementationAddress =
   "0x36963236d915e4e9b5f70677eBD1ea3e69Cfbbd6" as const;
 
-export async function getAccount(
+export function getAccount(
   tokenContract: string,
   tokenId: string,
-  provider: providers.BaseProvider
-) {
-  const registry = new Contract(
-    erc6551RegistryAddress,
-    erc6551RegistryAbi,
-    provider
-  );
-
-  const { chainId } = await provider.getNetwork();
-
-  return registry.callStatic.account(
+  chainId: number,
+): string {
+  const code = getCreationCode(
     erc6551AccountImplementationAddress,
     chainId,
     tokenContract,
     tokenId,
-    0
+    "0"
```

Comment:
It's safer to pass this as a string since uint256 is a BigNumber. We hardcode to "0" which makes it a null point at the moment, but in the future we'll probably allow passing of salts and we don't want to be forcing users to pass in BigNumbers or worse, a `number` that overflows.

Reply:
Similar thinking on L26. The salt hex will always be `0x00....00` but in the future we'll probably want allow passing in of the salt, and properly converting it to bytes32 hex.

Reply:
@johnnyshankman good call on this!